src/RooParametricHist.cxx

```diff
@@ -34,11 +34,7 @@ RooParametricHist::RooParametricHist(const char *name,
   pars("pars","pars",this)
   //SM_shape("SM_shape","SM_shape",this,_SM_shape),
 {
-  TIterator *varIter=_pars.createIterator();
-  RooAbsReal *fVar;
-  while ( (fVar = (RooAbsReal*)varIter->Next()) ){
-	pars.add(*fVar);
-  }
+  pars.add(_pars);
```

Comment:
This fixes a memory leak because the `TIterator *varIter` was not deleted originally.